### PR TITLE
I36 single demo

### PR DESCRIPTION
**Pull request by eyvorchuk**
This PR deals with [issue 36](https://github.com/pacificclimate/thunderbird/issues/36), which creates a single demo notebook showcasing the current wps processes in `thunderbird` with short descriptions on how they work.

### CODE REVIEW

**Comment by eyvorchuk** [comment]
> It is more likely though that this is coming from my end.

Sangwon just told me that he's also getting errors, so I made mistakes somewhere. I'll look into it.

File: notebooks/demo_processes.ipynb [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+     "output_type": "stream",
+     "text": [
+      "update_metadataResponse(\n",
+      "    output='http://localhost:5001/outputs/87b92afc-b663-11ea-afd0-c86000e3f2fd/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_copy.nc'\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "updates = resource_filename(\"tests\", \"metadata-conversion/simple-conversion.yaml\")\n",
```
Comment (sum1lim) [inline notebooks/demo_processes.ipynb:442]: I think this should be `data/metadata-conversion/simple-conversion.yaml`?

Comment (eyvorchuk) [reply notebooks/demo_processes.ipynb:442]: According to master, the path is `tests/metadata-conversion/simple-conversion.yaml`.

Comment (sum1lim) [reply notebooks/demo_processes.ipynb:442]: Also, I don't see `data` directory` in `venv/lib/python3.7/site-packages/tests` a virtual environment is set. @nikola-rados , do you have any idea on this?

Comment (eyvorchuk) [reply notebooks/demo_processes.ipynb:442]: That should be pointing to `thunderbird/tests/`...

Comment (nikola-rados) [reply notebooks/demo_processes.ipynb:442]: Hmm not sure @sum1lim I can see the data dir.

Comment (eyvorchuk) [reply notebooks/demo_processes.ipynb:442]: Do your pytests work?

Comment (eyvorchuk) [reply notebooks/demo_processes.ipynb:442]: Never mind `resource_filename("tests", "_")` changes depending on if your current directory is `notebooks` or `thunderbird`.

File: notebooks/demo_generate_prsn.ipynb [hunk 2]
```diff
@@ -15,17 +15,25 @@
...
   },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "import os \n",
+    "if os.path.basename(os.getcwd()) != \"thunderbird\": # Ensure current directory is always 'thunderbird' for 'resource_filename' to work\n",
+    "    os.chdir('../')"
```
Comment (sum1lim) [inline notebooks/demo_generate_prsn.ipynb:29]: This does resolve the current directory issue. Nice work!

File: notebooks/demo_processes.ipynb [hunk 3]
```diff
@@ -428,21 +436,23 @@
...
       "update_metadataResponse(\n",
-      "    output='http://localhost:5001/outputs/790e526a-b66e-11ea-afd0-c86000e3f2fd/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_copy.nc'\n",
+      "    output='http://localhost:5001/outputs/010c4f42-b6ff-11ea-afd0-c86000e3f2fd/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_copy.nc'\n",
       ")\n"
      ]
     }
    ],
    "source": [
     "updates = resource_filename(\"tests\", \"metadata-conversion/simple-conversion.yaml\")\n",
+    "print(updates)\n",
```
Comment (sum1lim) [inline notebooks/demo_processes.ipynb]: Probably want to delete the `print` statement. Checking the filepath is no longer our interest.

File: notebooks/demo_generate_prsn.ipynb (hunk 2 above)
Comment (eyvorchuk) [reply notebooks/demo_generate_prsn.ipynb:29]: Could you actually try running `demo_processes.ipynb` again? I pushed a commit that moves the individual demos to their own subdirectory and I'm not sure if I'll have to move up one more level as a result.

**Pull request merged**